contracts/finance/VestingWalletConfidentialFactory.sol

```diff
@@ -15,33 +15,19 @@ import {VestingWalletCliffConfidential} from "./VestingWalletCliffConfidential.s
  */
 abstract contract VestingWalletConfidentialFactory {
     struct VestingPlan {
-        address beneficiary;
         externalEuint64 encryptedAmount;
-        uint48 startTimestamp;
-        uint48 durationSeconds;
-        uint48 cliffSeconds;
+        bytes initialization;
     }
 
     address private immutable _vestingImplementation;
 
     event VestingWalletConfidentialFunded(
```

Comment:
Filtering events by `beneficiary` would be useful for developers.

Reply:
I think integrators can look at the initialization code